cumulusci/core/config/project_config.py

```diff
@@ -123,9 +124,13 @@ def _validate_config(self):
         self._validate_package_api_format()
 
     def _validate_package_api_format(self):
+        api_version_regex = r"^\d\d+\.0$"
         api_version = self.config["project"]["package"]["api_version"]
-        if not isinstance(api_version, str):
-            message = f"Package API Version must be specified as a string (i.e. '45.0'). Found: {type(api_version)}"
+
+        if not re.match(api_version_regex, api_version):
```

Comment:
I think this will throw an exception if api_version is not actually a string. Let's do `api_version = str(api_version)` to make sure it is one, then do the validation against the string and return it.